README.rst

```diff
@@ -40,14 +46,26 @@ Silicone is free software under a BSD 3-Clause License, see `LICENSE <https://gi
 Installation
 ------------
 
-Instructions to be written.
+This Python package can be installed directly from github. The release version is hosted at
```

Comment:
Let's skype about this. Best practice is to release like is done for [scmdata](https://scmdata.readthedocs.io/en/latest/development.html#releasing).

That means users can then install simply by doing `pip install silicone` or `conda install silicone`. If they want a specific version, they can specify e.g. `pip install silicone==0.2.1`. With releases on pypi (pip) and conda, we don't have to worry about a 'release version' and a 'development version'. I would suggest we simply do away with the znicholls/silicone (when you're ready) and move it all to GranthamImperial.

Reply:
This seems like a good idea, but to be done in a separate pull request. Currently what's written there is true, so I'm inclined to stick with that and update it later.